HISTORY.rst

```diff
@@ -48,6 +48,12 @@ Bug Fixes:
 
 Other Changes:
 
+* Update sponsoring section, add backers, thanks to `@hackebrot`_ (#950)
+* Changed link title for cookiecutter-django-paas, thanks to `@mpachas`_ (#933)
+* Merge pull request from stevepiercy/patch-6, thanks to `@hackebrot`_ (#911)
+* Fix Windows installation instructions, thanks to `@stevepiercy`_ (#887)
+* Fix the doc string in is_copy_only_path funciton for PEP257, thanks to `@harshtheworkman`_ (#742)
```

Comment:
742 is issue, not PR. What PR was mentioned here?
is_copy_path is in PR 996, already in history.